services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime = None
+) -> QuerySet | MovieSession:

```

Comment:
The return type hint `QuerySet | MovieSession` is misleading for the `get_movies_sessions` function. Since this function returns a `QuerySet`, the return type should be `QuerySet` only.